### DISCUSSION TITLE
Shortcut keys for saving annotation

### DISCUSSION THREAD

**Discussion by styw1234** [sumatrapdfreader/sumatrapdf#2455] [Q&A] [answered]
It is necessary to set shortcut keys for saving annotation, like Shift + Ctrl + S

or is there any possibility that the users are able to set shortcuts by themselves?

![微信截图_20220404200535](https://user-images.githubusercontent.com/9974018/161540100-badd023e-87a7-402c-8a4f-1bedff84aa2c.png)

**Comment by GitHubRulesOK** [comment]
Pre-release features don't usually have shortcut keys, nor some permanent features with single key like <kbd>C</kbd> or some without any like `show in folder`
however there are some added new exceptions for those in the context menu like <kbd>A</kbd> <kbd>U</kbd> <kbd>Del</kbd>

**Comment by kjk** [comment, marked as answer]
Makes sense, I've added `Ctrl-Shift-S` as shortcut for `Save annotations to existing PDF` in pre-release.

Note that this shortcut was used for `Save as link` functionality but it's undocumented and I assume rarely used. Nonetheless it is a behavior change.
